data-prepper-plugin-framework/src/test/java/org/opensearch/dataprepper/plugin/ExtensionLoaderTest.java

```diff
@@ -67,14 +69,21 @@ class ExtensionLoaderTest {
     private ArgumentCaptor<Collection<PluginError>> pluginErrorsArgumentCaptor;
     private PluginErrorCollector pluginErrorCollector;
 
+    private PluginCreatorContext pluginCreatorContext;
```

Comment:
You don't need this field. You can move it to a local variable in `setUp`. Also, you might not need it if you use a different Comparator.

Reply:
I think I will keep the same comparator.